Lib/types.py

```diff
@@ -293,6 +294,12 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
+def f(x: undefined):
+    pass
+
+AnnotationName = type(f.__annotations__["x"])
+del f
+
```

Comment:
This could be moved up to use the function `_f()` on line 12. Then both types could be extracted from one function instead of two.